lib/medical_records/user_eligibility/configuration.rb

```diff
@@ -19,7 +19,7 @@ class Configuration < Common::Client::Configuration::REST
     # @return [String] Base path for dependent URLs
     #
     def base_path
-      "#{Settings.mhv.medical_records.host}/v1/usermgmt/usereligibility/"
+      "#{Settings.mhv.api_gateway.hosts.usermgmt}/v1/usermgmt/usereligibility/"
```

Comment:
This points to a different upstream host (4426). Make sure that's the upstream port you want.

Reply:
Thanks @ericboehs -- that is indeed the host we're aiming for